array.go

```diff
@@ -59,7 +59,8 @@ func (a *Array) write(dst []byte) []byte {
 // Object marshals an object that implement the LogObjectMarshaler
 // interface and appends it to the array.
 func (a *Array) Object(obj LogObjectMarshaler) *Array {
-	a.buf = appendObject(enc.AppendArrayDelim(a.buf), obj)
+	// TODO can we get context, stack, and hooks here?
```

Comment:
@rs not sure how (or if it's reasonable) to get this information for Array.Object